storage/indexer/src/schema/transaction_by_account/mod.rs

```diff
@@ -0,0 +1,67 @@
+// Copyright © Aptos Foundation
```

Comment:
I have to say the duplication is a bit too much.

Extracting to a core-index-schemas crate and make the schemas public can be nice.